src/west/app/main.py

```diff
@@ -154,36 +152,48 @@ def handle_builtin_manifest_load_err(self, args):
         # Handle ManifestVersionError is a special case.
         if isinstance(self.mle, ManifestVersionError):
             if args.command == 'help':
-                log.wrn(mve_msg(self.mle, suggest_upgrade=False) +
-                        '\n  Cannot get extension command help, ' +
-                        "and most commands won't run." +
-                        '\n  To silence this warning, upgrade west.')
+                self.queued_io.append(
```

Comment:
Have you considred adding something that automates the whole `self.queued_io.append( lambda)` part?

Reply:
Yeah, I couldn't think of anything that would make it simpler. We want to have the real command instance when we do the I/O so we can include WestCommand instance data in there. That implies a need for a new scope, which we can only do with a `lambda` or a `def`. So that's got the stay the way it is IMO.

We could add `self.queue_io(lambda cmd: ...)` to save a bit of typing, but I think the extra level of indirection is less clear, not more clear.

Reply:
Understood, thanks for the explanation!